### DISCUSSION TITLE
Board support package flash to Jetson orin nx

### DISCUSSION THREAD

**Discussion by yusufcaglayann** [OE4T/meta-tegra#1429] [Q&A] [answered]
I am installing bsp for the first jetson orin nx installation. After typing the ./install.sh -create_default_account command, the process goes smoothly for a while, and I get the following error in step 2. What is the error and how can i fix it?

************************************************ * * * Step 2: Boot the device with flash initrd image * * * ************************************************ ~/JetPack_5.1.1_Linux_JETSON/Linux_for_Tegra/temp_initrdflash/bootloader0 ~/JetPack_5.1.1_Linux_JETSON/Linux_for_Tegra ./tegraflash.py --bl uefi_jetson_with_dtb_sigheader.bin.encrypt --bct br_bct_BR.bct --securedev --bldtb tegra234-p3767-0000-p3509-a02.dtb --applet rcm_2_encrypt.rcm --applet_softfuse rcm_1_encrypt.rcm --cmd “rcmboot” --cfg secureflash.xml --chip 0x23 --mb1_bct mb1_bct_MB1_sigheader.bct.encrypt --mem_bct mem_rcm_sigheader.bct.encrypt --mb1_cold_boot_bct mb1_cold_boot_bct_MB1_sigheader.bct.encrypt --mb1_bin mb1_t234_prod_aligned_sigheader.bin.encrypt --psc_bl1_bin psc_bl1_t234_prod_aligned_sigheader.bin.encrypt --mem_bct_cold_boot mem_coldboot_sigheader.bct.encrypt --bins “psc_fw pscfw_t234_prod_sigheader.bin.encrypt; mts_mce mce_flash_o10_cr_prod_sigheader.bin.encrypt; mb2_applet applet_t234_sigheader.bin.encrypt; mb2_bootloader mb2_t234_with_mb2_cold_boot_bct_MB2_sigheader.bin.encrypt; xusb_fw xusb_t234_prod_sigheader.bin.encrypt; dce_fw display-t234-dce_sigheader.bin.encrypt; nvdec nvdec_t234_prod_sigheader.fw.encrypt; bpmp_fw bpmp_t234-TE990M-A1_prod_sigheader.bin.encrypt; bpmp_fw_dtb tegra234-bpmp-3767-0001-3509-a02_with_odm_sigheader.dtb.encrypt; sce_fw camera-rtcpu-sce_sigheader.img.encrypt; rce_fw camera-rtcpu-t234-rce_sigheader.img.encrypt; ape_fw adsp-fw_sigheader.bin.encrypt; spe_fw spe_t234_sigheader.bin.encrypt; tos tos-optee_t234_sigheader.img.encrypt; eks eks_t234_sigheader.img.encrypt; kernel boot0.imgimg; kernel_dtb tegra234-p3767-0001-p3509-a02.dtb” --secondary_gpt_backup --bct_backup --boot_chain A --instance 1-1 Welcome to Tegra Flash version 1.0.0 Type ? or help for help and q or quit to exit Use ! to execute system commands Entering RCM boot [ 0.0202 ] mb1_t234_prod_aligned_sigheader.bin.encrypt filename is from --mb1_bin [ 0.0202 ] psc_bl1_t234_prod_aligned_sigheader.bin.encrypt filename is from --psc_bl1_bin [ 0.0202 ] rcm boot with presigned binaries [ 0.0210 ] tegrarcm_v2 --instance 1-1 --new_session --chip 0x23 0 --uid --download bct_br br_bct_BR.bct --download mb1 mb1_t234_prod_aligned_sigheader.bin.encrypt --download psc_bl1 psc_bl1_t234_prod_aligned_sigheader.bin.encrypt --download bct_mb1 mb1_bct_MB1_sigheader.bct.encrypt [ 0.0217 ] BR_CID: 0x80012344705DD4614000000010FE8080 [ 0.0269 ] Sending bct_br [ 0.0308 ] Sending mb1 [ 0.0318 ] Sending psc_bl1 [ 0.0524 ] Sending bct_mb1
[ 0.0649 ] Generating blob for T23x
[ 0.0692 ] tegrahost_v2 --chip 0x23 0 --generateblob blob.xml blob.bin
[ 0.0755 ] Stat for blob_boot0.imgimg failed
[ 0.0755 ]
Error: Return value 19
Command tegrahost_v2 --chip 0x23 0 --generateblob blob.xml blob.bin
Cleaning up…

**Comment by dwalkes** [comment, marked as answer]
Hi @yusufcaglayann it looks like you are using some install instructions based on NVIDIA's Jetpack distribution as opposed to the OE4T yocto based root filesystem build process used by this project, is that correct?

The instructions for this project are at https://github.com/OE4T/meta-tegra/wiki/Flashing-the-Jetson-Dev-Kit or https://github.com/OE4T/meta-tegra/wiki/initrd-flashing-support and don't use the `install.sh` command you are referencing.  I suspect you may need to move your question to a different forum.

**Comment by Kaangok** [comment]
Hi Yusuf, I'm having the same issue. I have the reComputer J401 carrier board. I'd appreciate it if you could help.

**Reply by yusufcaglayann** [reply]
Which ubuntu version are you using during installation?

**Reply by Kaangok** [reply]
I'm trying to flash JetPack 6.1 onto my Orin NX 16GB module, which is installed on a reComputer J401 carrier board. I also have a Samsung 990 PRO SSD.

I was previously using Dragon OS (based on Ubuntu 22.04.5 LTS) on my host machine to flash JetPack, but I encountered errors. I have since switched to Ubuntu 24.04.1 LTS, but I'm still getting the same error. What can I do to resolve this issue? Do you know what might be causing it?

**Reply by yusufcaglayann** [reply]
I used ubuntu 18.04 as host machine during the first installation and I encountered the error mentioned above. Then I switched to Ubuntu 20.04 and the installation ended successfully.

**Reply by Kaangok** [reply]
Hi yusufcaglayann,
Thank you for your help! Your solution worked, and the flashing process is currently underway.
I appreciate your support.
Sincerely,

**Reply by yusufcaglayann** [reply]
You're welcome. I am happy to help you with that.
